Zend/zend_vm_gen.php

```diff
@@ -2497,7 +2497,7 @@ function gen_vm($def, $skel) {
             if (ZEND_VM_KIND == ZEND_VM_KIND_GOTO
              || ZEND_VM_KIND == ZEND_VM_KIND_SWITCH
              || (ZEND_VM_KIND == ZEND_VM_KIND_HYBRID && $hot)) {
-                foreach (explode(",", $param) as $p) {
+                foreach (explode(",", $param ?: '') as $p) {
```

Comment:
It would make more sense to me to skip the foreach entirely if `$param` is `null`.

Reply:
Done.